internal/core/services/room_manager/room_manager.go

```diff
@@ -37,15 +38,17 @@ type RoomManager struct {
 	roomStorage     ports.RoomStorage
 	instanceStorage ports.GameRoomInstanceStorage
 	runtime         ports.Runtime
+	configs         config.Config
 }
 
-func NewRoomManager(clock ports.Clock, portAllocator ports.PortAllocator, roomStorage ports.RoomStorage, instanceStorage ports.GameRoomInstanceStorage, runtime ports.Runtime) *RoomManager {
+func NewRoomManager(clock ports.Clock, portAllocator ports.PortAllocator, roomStorage ports.RoomStorage, instanceStorage ports.GameRoomInstanceStorage, runtime ports.Runtime, configs config.Config) *RoomManager {
```

Comment:
Until now, we were avoiding having references to the "configuration layer" on the core/adapters (like [here](https://github.com/topfreegames/maestro/blob/next/internal/service/adapters.go#L97)) but since we didn't discuss this topic too much in-depth, what's your input on this?

My thoughts: By doing this, it seems to me that we're starting to attach too much of the service to the "infrastructure" the commands are using. Having a service initializer that takes each configuration attribute doesn't seem to scale that well. Still, alternatively, we can have a `RoomManagerOptions` struct to rely on instead of the configuration itself.

Reply:
Good idea, actually, devs on project manager are converting the entire YAML configuration to a struct and passing to the service just the scoped piece. We could follow the same approach, WDYT?

In any case, I would recommend opening a tech debt for that once we use the configuration in all over the code.

Reply:
@gabrielcorado @luizmiranda7 In this case, the first solution that comes to mind is creating a new port (e.g  RoomManagerOptions or RoomManagerConfig) with its respective adapter to maintain the architecture layers isolated

Reply:
Not sure if we require having one port for each of our services. Also, the configuration struct could live in this package and be initialized by our "infrastructure" code (like we do in the case of the adapter). WDYT?